### ISSUE TITLE
Flake8 linter is missing a usage of typing.List

### ISSUE THREAD

**Issue by sargunv** [markstory/lint-review#222]
https://github.com/PokeAPI/ditto/pull/17/files/5dd1caa407d9f0690678e697153334a2acc86d54#diff-f74be92c3a1504ba9272652dedc9cccaR2

You can see it commented saying "F401 'typing.List' imported but unused", but the usage is right there on line 17. Running flake8 locally doesn't give this error.

**Comment by markstory** [comment]
Could this be related to the issues you're having with config files? This project and by extension stickler run flake8 with request options and process the output.

**Comment by sargunv** [comment]
Perhaps, although in this specific run other config options seemed to be applied (I had them duplicated in the .stickler.yml instead of passing the config parameter)

Could it be something related to Python or flake8 version? Typing is actually a somewhat recent addition.

**Comment by markstory** [comment]
What version of python and flake8 are you using? By default stickler runs in python2, and flake8 was last updated 2 or 3 weeks ago.

**Comment by sargunv** [comment]
I'm using flake 3.5 and python 3.6.  My stickler config looks like this:

``` yaml
linters:

  black:
    config: ./pyproject.toml
    fixer: true

  flake8:
    fixer: true
    python: 3
    
    # copied from tox.ini
    max-complexity: 15
    ignore: W,E
    exclude: .venv, .git, __pycache__, dist, data

fixers:
  enable: true

```

**Comment by sargunv** [comment]
Still seeing this issue, see https://github.com/PokeAPI/ditto/pull/36 for another example, this time with `typing.Dict`

**Comment by sargunv** [comment]
Nope, it's working for some usages of type hints. It's only missing type hints on local variables (it sees hints on other things), and only when I run through Stickler, not when I run flake8 manually. 

After looking in to it, it appears this was a bug in pyflakes 1.5.0 and was fixed in 1.6.0. I can't find any reference to a pyflakes version in the repo, perhaps you'll have to require the updated version.

**Comment by markstory** [comment]
@sargunv I will bump up flake8 and pyflakes versions.
